src/plugins/discover/public/utils/truncate_styles.ts

```diff
@@ -32,18 +38,7 @@ const buildStylesheet = (maxHeight: number) => {
   ];
 };
 
-const flushThemedGlobals = () => {
```

Comment:
Regarding `dscTruncateByHeight:before` style above, is it working as expected? I can't see any gradient in when content is truncated in the legacy view.

Reply:
Here I just did not have a look what it's doing, thx, will check

Reply:
applied this suggested change

Reply:
@kertal Sure, thanks for checking!